save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/TimelineComponent.kt

```diff
@@ -28,45 +30,50 @@ val timelineComponent: FC<TimelineComponentProps> = FC { props ->
                 label = "Add date",
                 style = "secondary",
                 isOutline = true,
-                classes = "btn btn-primary"
+                classes = "btn btn-sm btn-primary"
             ) {
                 onClickCallback()
             }
         }
 
-        if (props.dates.isNotEmpty()) {
+        div {
+            className = ClassName("p-0 timeline-container")
             div {
-                className = ClassName("p-0 timeline-container")
+                className = ClassName("steps-container")
                 div {
-                    className = ClassName("steps-container")
-                    div {
-                        className = ClassName("line")
-                    }
-                    props.dates.toList()
-                        .sortedBy { it.second }
-                        .forEach { (label, dateTime) ->
-                            div {
-                                className = ClassName("step $hoverable")
+                    className = ClassName("line")
+                }
+                props.dates
+                    .plus(
+                        VulnerabilityDateType.SUBMITTED.value to
+                                (props.vulnerability.creationDateTime ?: LocalDateTime(0, 1, 1, 0, 0, 0, 0))
```

Comment:
Do we want such a placeholder or it is better to hide `SUBMITTED` date?

Reply:
No, not submitted, we would like to have a timeline always. 
A different point is that we need to move it to constant, I guess